rebar.config

```diff
@@ -49,7 +49,7 @@
 {plugins, [{swagger_endpoints, {git, "https://github.com/aeternity/swagger_endpoints", {tag, "0.2.1"}}}, {pc, "1.6.0"}]}.
 {swagger_endpoints, [{src, "config/swagger.yaml"}, {dst, "apps/aeutils/src/endpoints.erl"}]}.
 
-{relx, [{release, { epoch, "version value comes from VERSION" },
+{relx, [{release, { epoch, {cmd, "cat VERSION | tr -d '[:space:]'"}},
```

Comment:
I see this is implemented at https://github.com/erlware/relx/blob/v3.24.4/src/rlx_config.erl#L347

@tsloughter Is that cmd option documented? I did not find it documented in the erlware/relx repo readme or wiki. (My concern is that if undocumented it may change in the future abruptly.)

Reply:
It isn't. I haven't really been doing relx docs separate from rebar3 so for now I'll just be adding it to the docs here http://www.rebar3.org/docs/releases today.